common/static/js/xblock/core.js

```diff
@@ -69,7 +69,7 @@
         } else {
             block = {};
         }
-        block.element = element;
+        block.element = $element;
```

Comment:
This line has caused a regression: https://openedx.atlassian.net/browse/TNL-4161

What was the rationale for this specific change?